src/commit-subject.ts

```diff
@@ -5,29 +5,37 @@ import {MessageKey} from './message-key'
 import {QueueName} from './queue-name'
 
 /* The first line of a commit message.
- * Format: {COMMIT_SUBJECT_PREFIX}{MESSAGE_KEY}: {QUEUE_NAME}: job.ref.{COMMIT_HASH}
- * Example: 📝✅: queue_name: job.ref.1e31b549c630f806961a291b4e3d4a1471f37490
+ * Format: {COMMIT_SUBJECT_PREFIX}{MESSAGE_KEY}: {QUEUE_NAME}: job.{JOB_ID} job.ref.{COMMIT_HASH}
+ * Example: 📝✅: queue_name: job.42 job.ref.1e31b549c630f806961a291b4e3d4a1471f37490
  */
 export class CommitSubject {
   private messageKey: MessageKey
   private queueName: QueueName
   private jobRef: CommitHash
+  private id: number
 
   constructor(
     messageKey: MessageKey,
     queueName: QueueName,
-    jobRef: CommitHash
+    jobRef: CommitHash,
+    id: number
```

Comment:
@yeraydavidrodriguez I would order them in the order they appear in the commit subject :-). I mean I would put the job id before the job ref. And remember to change the type from `number` to `JobId`.

Reply:
You are right. Applied.